core/commands/log.go

```diff
@@ -50,3 +68,64 @@ output of a running daemon.
 	},
 	Type: MessageOutput{},
 }
+
+var logReadCmd = &cmds.Command{
+	Helptext: cmds.HelpText{
+		Tagline: "Read the logs",
+		ShortDescription: `
+'ipfs log read' is a utility command used to read the log output.
+
+By default, the last 50 lines are returned. For more, use '-n=<number of lines>'.
+Use '--stream' (or '-s') to stream the current output (in addition to the '-n' lines).
```

Comment:
Let's rename this to `tail` and `tail -f` (to follow) as users are used to that.

```
ipfs log tail
ipfs log tail -f
```